docs/chaos/index.md

```diff
@@ -7,6 +7,40 @@
 
 ![Simmy](../media/simmy-logo.png)
 
+## Usage
+
+<!-- snippet: chaos-usage -->
+```cs
+var builder = new ResiliencePipelineBuilder<HttpResponseMessage>();
+
+// First, configure regular resilience strategies
+builder
+    .AddConcurrencyLimiter(10, 100)
+    .AddRetry(new())
+    .AddCircuitBreaker(new())
```

Comment:
Should we add a parameterless overload for both these methods, if such usage is valid? Otherwise this just looks a bit bleurgh.

If doing that isn't valid, then we should change these to something like `new() { /* configure options */ }`.

Reply:
I have used `new() { /* configure options */ }`. The actual configuration of regular resilience strategies is not important here and I want to focus on chaos strategies here.